source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveCardRenderer.cs

```diff
@@ -839,6 +842,129 @@ protected static HtmlTag TextBlockRender(AdaptiveTextBlock textBlock, AdaptiveRe
             return uiTextBlock;
         }
 
+        protected static HtmlTag RichTextBlockRender(AdaptiveRichTextBlock richTextBlock, AdaptiveRenderContext context)
+        {
+            var uiTextBlock = new HtmlTag("div", false)
+                .AddClass($"ac-{richTextBlock.Type.Replace(".", "").ToLower()}")
+                .Attr("name", richTextBlock.Id)
+                .Style("box-sizing", "border-box")
+                .Style("text-align", richTextBlock.HorizontalAlignment.ToString().ToLower());
+
+            if (richTextBlock.Height == AdaptiveHeight.Stretch)
+            {
+                uiTextBlock.Style("flex", "1 1 100%");
+            }
+
+            if (!richTextBlock.IsVisible)
+            {
+                uiTextBlock.Style("display", "none");
+            }
+
+            var setWrapStyleOnParagraph = false;
+            if (richTextBlock.Wrap == false)
```

Comment:
this feels a bit awkward

Reply:
This was all copied over from the TextBlockRenderer. Just replaced it with the ternary operator and got rid of the extra boolean in both.

---
In reply to: [263163649](https://github.com/Microsoft/AdaptiveCards/pull/2519#discussion_r263163649) [](ancestors = 263163649)